t/05-draft4.t

```diff
@@ -15,19 +15,29 @@ my @tests = map { glob $_ } qw(
     t/test-suite/tests/draft4/optional/*.json
 );
 
+# following tests fail because Valiemon treats 1 as string:
+# - not.json
+# - type.json
+# - maxLength.json
+# - minLength.json
+# - anyOf.json
+# - oneOf.json
+# following tests fail because patternProperties are not implemented:
```

Comment:
:point_up: Incorrect.

`patternProperties` will be the last feature that is necessary, implementable, but unimplemented.

(If we need to implement all of necessary features, we need to support multiple scope!!)